### ISSUE TITLE
how to collect the raw microphone data during recording ? 

### ISSUE THREAD

**Issue by woosungchu** [nstudio/nativescript-audio#93]
While I try to make my previous cordova project to nativescript style, which can analyze 
raw audio data in real-time, this open-source help me a lot.

But I think it hasn't proper API to collect  the raw microphone data
like .. [ 0.003434 , -0.12355 , 0.0002323 ....... ] this.

Can you answer me if it have API or not?

**Comment by bradmartin** [comment]
I don't think so but if you can point me to a Cordova implementation (I assume this is a cordova plugin you've used) I can check the native APIs being used and see what needs to be addressed with this plugin. Thanks

**Comment by woosungchu** [comment]
this demo is what I used.
https://github.com/edimuj/cordova-plugin-audioinput/blob/master/demo/file-demo.js

**evt.data** will return raw microphone data ( [ 0.003434 , -0.12355 , 0.0002323 ....... ] )

```
function onAudioInputCapture(evt) {
    try {
        if (evt && evt.data) {
            // Increase the debug counter for received data
            totalReceivedData += evt.data.length;
console.log(evt.data)
            // Add the chunk to the buffer
            audioDataBuffer = audioDataBuffer.concat(evt.data);
        }
    }
    catch (ex) {
        alert("onAudioInputCapture ex: " + ex);
    }
}
```

Or you can just clone and follow readme.md file 
https://github.com/woosungchu/corodova-recording
